src/test/java/com/endava/internship/collections/StudentListTest.java

```diff
@@ -209,9 +210,33 @@ void tesGetShouldBeOkWithNulls() {
         //then
         assertEquals(null,returnedStudent);
     }
+    @ParameterizedTest
+    @ValueSource(ints = {-10,1,2,7,89,100})
+    void setThrowsExceptionInvalidIndex(int index) {
+        //given
+        Student petea =new Student(names[3],birthdays[2],details[3] );
```

Comment:
will be more understandable to use variables instead of array

Reply:
Fixed!